micromanager_gui/main_window.py

```diff
@@ -275,6 +276,10 @@ def _on_mda_started(self, sequence: useq.MDASequence):
             # originated from user script - assume it's an mda
             self._mda_meta.mode = "mda"
 
+        if self._mda_meta.mode == "explorer":
+            with contextlib.suppress(ValueError):
+                self.viewer.layers.remove("preview")
+
```

Comment:
Why do this?

Reply:
What happens if you do explorer and then start live mode - does the same problem crop up again?

Reply:
The microscope is not free at the moment, I need to check this better probably...I will record a video and show you what I mean. My point is that since napari has a nice world coordinates system, I'd love to use it to map the microscope stage...

Reply:
> My point is that since napari has a nice world coordinates system, I'd love to use it to map the microscope stage...

Absolutely! I just didn't understnad the issue at first. That sounds maybe complicated to get right - perhaps best to open an issue dedicated tothis?

> Anyway, I reverted the last change, let's leave this PR only for the mda.cancel() method

that keeps this nice and simple. This is good to go from me